thrift/history.thrift

```diff
@@ -644,6 +653,7 @@ service HistoryService {
       5: shared.DomainNotActiveError domainNotActiveError,
       6: shared.LimitExceededError limitExceededError,
       7: shared.ServiceBusyError serviceBusyError,
+      8: shared.WorkflowExecutionAlreadyCompletedError workflowExecutionAlreadyCompletedError,
```

Comment:
I don't think ResetWorkflowExecution will return this error? It's ok to reset a completed workflow.